code/modules/gear_presets/survivors/solaris/crashlanding-offices_insert_bigred.dm

```diff
@@ -120,10 +139,10 @@
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/head/manager(new_human), WEAR_HEAD)
 	new_human.equip_to_slot_or_del(new /obj/item/reagent_container/glass/beaker/vial/random/good(new_human), WEAR_IN_JACKET)
 	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/firstaid/ert(new_human), WEAR_R_STORE)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/survival/full(new_human), WEAR_L_STORE)
 	add_pmc_survivor_weapon(new_human)
 	add_random_cl_survivor_loot(new_human)
 
-	..()
 // only used on the spawner of all of those above...
 /datum/equipment_preset/synth/survivor/pmc
```

Comment:
Missed changing faction_group on this preset.

Reply:
I did this on purpose both here and above at line 42 to prevent any of the roles from inheriting the default spawn equipment (generic colony pistols, improvised firebombs, etc). This was done to give the medic room for his ammo to spawn in his bag and because it didn't make sense for them to have this equipment flavor-wise as they have all just woken up from a crash. I am not familiar with the code so if the way I do it has unintended side-effects or there is a better way of doing what I want please let me know.